helm/cas-airflow/templates/routes/route.yaml

```
@@ -0,0 +1,22 @@
+kind: Route
+apiVersion: route.openshift.io/v1
+metadata:
+  name: {{ .Release.Name }}-web
+spec:
+  host: {{ .Release.Name }}-web-{{ .Release.Namespace }}.apps.silver.devops.gov.bc.ca
```

Something like this would be a bit better (need to add a helper function and a new value): 
```suggestion
  host: {{ .Release.Name }}-{{ include "cas-airflow.namespaceSuffix" . }}.{{ .Values.route.clusterDomain }}
```